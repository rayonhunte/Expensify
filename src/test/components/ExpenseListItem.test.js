import React from 'react';
import {shallow} from 'enzyme';
import ExpensesListItem from '../../components/ExpenseListItem';


const expense = {
  id: '1',
  description: 'Gum',
  note: '',
  amount: 195,
  createAt: 0
}


test('should render expense item',()=>{
  const wrapper = shallow(<ExpensesListItem expense={expense}/>);
  expect(wrapper).toMatchSnapshot();
});