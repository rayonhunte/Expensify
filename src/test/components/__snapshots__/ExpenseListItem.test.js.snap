// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render expense item 1`] = `
<Link
  className="list-item"
  replace={false}
  to="/edit/undefined"
>
  <div>
    <h3
      className="list-item__title"
    />
    <span
      className="list-item__sub-title"
    >
      December 31st, 1969
    </span>
  </div>
  <div>
    <h3
      className="list-item__data"
    >
       
      $0.0
       
    </h3>
  </div>
</Link>
`;
